rskj-core/src/main/java/co/rsk/cli/tools/DbMigrate.java

```diff
@@ -183,12 +184,21 @@ private void migrate(DbMigrationInformation dbMigrationInformation) {
 
         logger.info("Migrating data source: {}", sourceKeyValueDataSource.getName());
 
-        sourceKeyValueDataSource.keys().forEach(sourceKey ->
+        try (DataSourceKeyIterator iterator = sourceKeyValueDataSource.keyIterator()) {
+            iterator.seekToFirst();
```

Comment:
Fo the DBIterator instance I think it is not needed and for the RocksDbIterator I think it could be done internally as it has no any extra utility outside in our use case.

Reply:
Addressed